frontend/src/models/groupsModel.ts

```diff
@@ -4,12 +4,12 @@ import { GroupType } from '~/types'
 import { teamLogic } from 'scenes/teamLogic'
 import { groupsModelType } from './groupsModelType'
 import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
-import { groupsAccessLogic } from 'lib/introductions/groupsAccessLogic'
+import { groupsAccessLogic, GroupsAccessStatus } from 'lib/introductions/groupsAccessLogic'
 
 export const groupsModel = kea<groupsModelType>({
     path: ['models', 'groupsModel'],
     connect: {
-        values: [teamLogic, ['currentTeamId'], groupsAccessLogic, ['groupsEnabled']],
+        values: [teamLogic, ['currentTeamId'], groupsAccessLogic, ['groupsEnabled', 'groupsAccessStatus']],
```

Comment:
Unsure if this selector (`groupsAccessStatus`) is intended only for upsells, but if it does control display logic, we seem to have a small bug (even though it wasn't changed in this PR). If `preflight?.instance_preferences?.disable_paid_fs` we automatically set groups as hidden. Yet, technically a user could have set this flag but still have a valid license for groups.

Reply:
That's already handled in `showGroupsOptions`.

I will follow up with a different name for `groupsAccessStatus` as it's causing confusion, though the underlying tension will remain due to complicated business logic being involved.